ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1.admin;
+
+option java_outer_classname = "UserManagementServiceOuterClass";
+option java_package = "com.daml.ledger.api.v1.admin";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1.Admin";
+
+import "google/protobuf/empty.proto";
+
+
+// Experimental API to manage users and their rights for interacting with the Ledger API
+// served by a participant node.
+//
+// The API roughly follows the Google API style guidelines for resource-oriented design
+// (https://cloud.google.com/apis/design/resources) with the following simplifications to
+// ease its implementation:
+//
+// 1. List methods do not (yet) support pagination,
+//    as we expect to have fewer than 10k users and 1k rights per user.
+//
+// 2. Resources are not named as per the Google API style guide,
+//    as this would be in contrast with the other Ledger API services.
+//
+// Authorization rules for the services RPCs are specified on the `RpcNameRequest`
+// messages as boolean expressions over these facts:
+// - ``HasRight(r)``: true iff the authenticated user has right `r`
+// - ``IsAuthenticatedUser(u)``: true iff ``u`` is equal to the name of the authenticated user
+//
+service UserManagementService {
+
+    // Create a new user, failing if it already exists.
+    rpc CreateUser (CreateUserRequest) returns (User);
+
+    // Get the user data of a specific user or the authenticated user.
+    rpc GetUser (GetUserRequest) returns (User);
+
+    // Delete an existing user and all its rights.
+    rpc DeleteUser (DeleteUserRequest) returns (google.protobuf.Empty);
```

Comment:
I would recommend against using `Empty` as the response type. A specific message that doesn't have any fields is as "expensive" as `Empty`, but allows you to add fields later without breaking the API.

Reply:
I can do that. It's against the style guide here: https://cloud.google.com/apis/design/standard_methods, but it is a light violation, and can come in handy in future evoluations.

Reply:
Addressed in https://github.com/digital-asset/daml/pull/11908